### DISCUSSION TITLE
Did OpenSea support disappear in latest build for you?

### DISCUSSION THREAD

**Discussion by jeff-luszcz** [jvde-github/AIS-catcher#103] [General]
I saw it in the build I made from source yesterday, but then when I rebuilt using abcd567a's install script I'm not seeing it available any more in the built in web page.

<img width="319" alt="Screen Shot 2023-05-03 at 6 04 05 PM" src="https://user-images.githubusercontent.com/8672214/236060376-c82c8455-9fe2-4ea4-8be7-7dc7893fd78b.png">

Using version v0.45e-112-g00897e5

**Comment by jvde-github** [comment]
I suspect that this install script includes all example plugins by default including [maps_with_no_labels.pjs](https://github.com/jvde-github/AIS-catcher/blob/main/plugins/maps_with_no_labels.pjs). I would remove these plugins from the plugin directory and only include the ones that you want.

Have not tested but I suspect that is the issue. Note you can click 
![image](https://user-images.githubusercontent.com/52420030/236062088-187dc7b2-f33f-46c9-a54a-a31057aaafd5.png)
to see the plugins that are included.

Hope that helps

**Reply by jvde-github** [reply]
Note that once you have removed the mentioned plugin, you need to restart AIS-catcher to make the change effective.

**Comment by jeff-luszcz** [comment]
Perfect thanks for the quick response. I changed the plugin name from maps_with_no_labels.pjs.bak to maps_with_no_labels.pjs.bak and stoped and restarted the server and the opensea map option is visible again as expected!

**Reply by Taxom** [reply]
I recommend renaming your folder with plugins. And change it in the config (example: -N PLUGIN_DIR /usr/share/aiscatcher/plugins
). Then, at the next time updating, this script, your plugins will remain not touched. Just do not forget to choose to keep the configuration file when updating.
